src/main/kotlin/uk/gov/justice/digital/hmpps/nomisprisonerapi/movements/TemporaryAbsenceApplicationResponse.kt

```diff
@@ -0,0 +1,72 @@
+package uk.gov.justice.digital.hmpps.nomisprisonerapi.movements
+
+import io.swagger.v3.oas.annotations.media.Schema
+import uk.gov.justice.digital.hmpps.nomisprisonerapi.helpers.NomisAudit
+import java.time.LocalDate
+import java.time.LocalDateTime
+
+@Schema(description = "Temporary absence application response")
+data class TemporaryAbsenceApplicationResponse(
+  @Schema(description = "Booking ID")
+  val bookingId: Long,
+
+  @Schema(description = "Movement application ID")
+  val movementApplicationId: Long,
+
+  @Schema(description = "Event sub type")
+  val eventSubType: String,
+
+  @Schema(description = "Application date")
+  val applicationDate: LocalDateTime,
```

Comment:
It is also common to give this the name `applicationDatetime` - but I'm not too bothered